### PR TITLE
[docs][zh-cn] synced recent updates

### PR DESCRIPTION
**Pull request by Jinjiang**
Ref: https://github.com/vuejs/vue-test-utils/compare/865c526...6fa6e06
/ping @fnlctrl 
Thanks.

### CODE REVIEW

**Comment by eddyerburgh** [comment]
Thanks @Jinjiang . Could you also remove Chai references from the chinese docs 🙏, like in this PR—https://github.com/vuejs/vue-test-utils/pull/439

**Comment by Jinjiang** [comment]
No problem.👌
Will do that soon.

File: docs/zh-cn/guides/using-with-vue-router.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ shallow(Component, {
 })
 ```
 
+> **注意：**在一个 `localVue` 上安装 Vue Router 时也会将 `$route` 和 `$router` 作为两个只读属性添加给该 `localVue`。这意味着如果你使用安装了 Vue Router 的 `localVue`，则不能在挂在一个组件时使用 `mocks` 选项来复写 `$route` 和 `$router`。
```
Comment (fnlctrl) [inline docs/zh-cn/guides/using-with-vue-router.md:21]: 复写 => 覆写

**Review by fnlctrl — APPROVED** [review]

File: docs/zh-cn/guides/using-with-vue-router.md (hunk 1 above)
Comment (Jinjiang) [reply docs/zh-cn/guides/using-with-vue-router.md:21]: @fnlctrl 我检查了一下，这个词我在别的一些地方也用了“复写” (如 config.md、optinons.md 还有 vue-cli、style-guide 等地方)，另外我的输入法、百度关键字推荐等地方似乎都只有“复写”这个词。这里是否可以保持“复写”？🙂
谢谢

**Comment by Jinjiang** [comment]
@eddyerburgh @fnlctrl Btw, seems this PR has been reviewed and is close to be approved, I'd like to create another PR separately for removing Chai ref if you two don't mind :-)

**Comment by eddyerburgh** [comment]
Yep that's fine, thanks @Jinjiang @fnlctrl 🙌

File: docs/zh-cn/guides/using-with-vue-router.md (hunk 1 above)
Comment (fnlctrl) [reply docs/zh-cn/guides/using-with-vue-router.md:21]: 抱歉这两天没看通知:joy:

复写: 把复写纸夹在两张或几张纸之间书写,能写成一式多份(http://www.zdic.net/c/d/152/339032.htm, https://baike.baidu.com/item/%E5%A4%8D%E5%86%99)
英语中对应的是Copywriting
image

汉语词典本身没有“覆写”这个词，"覆"也没有"复"常用，所以输入法没有收录。但鉴于over是覆盖的意思，用"覆"可能更为恰当
image

Comment (Jinjiang) [reply docs/zh-cn/guides/using-with-vue-router.md:21]: 认同。感谢指正，我稍后全局替换一下。

Comment (Jinjiang) [reply docs/zh-cn/guides/using-with-vue-router.md:21]: #444

**Pull request merged**